app/src/main/java/eu/kanade/tachiyomi/ui/browse/BrowseController.kt

```diff
@@ -115,21 +115,30 @@ class BrowseController :
 
         // SY -->
         private val tabTitles = (
-            if (preferences.latestTabInFront().get()) {
```

Comment:
At this point you might as well use a lamba

ie
`
private val tabTitles = {
code here, make a list and add to it based on the preferences
}()
`

Reply:
sorry i'm not very familiar with lambda function yet

Reply:
No problem